testing/tests/DevExpress.ui.events/eventsInteraction.tests.js

```diff
@@ -602,6 +613,33 @@ QUnit.test('gesture should be started with wrong direction after timeout', funct
     assert.equal(swipeFired, 1, 'swipestart was fired');
 });
 
+QUnit.test('gesture should be started with wrong direction after timeout, immediateTimeout: 100', function(assert) {
+    let swipeFired = 0;
+    const $element = $('#element').on(swipeEvents.start, { immediate: true, immediateTimeout: 100, direction: 'horizontal' }, function() {
+        swipeFired++;
+    });
+    const pointer = pointerMock($element);
+
+    pointer.start().down().move(0, 1);
+    assert.strictEqual(swipeFired, 0, 'swipestart was fired');
+    this.clock.tick(90);
+    assert.strictEqual(swipeFired, 0, 'swipestart was not fired');
+    this.clock.tick(10);
+    pointer.move(0, 1);
```

Comment:
y do we need this additional move? Seems like swipe event should be fired just after 100ms timeout without additional actions

Reply:
Yes, we need to do it to initiate new action that will be passed